apps/torus-portal/src/app/(pages)/(permission-graph)/_components/graph-sheet/graph-sheet-details/graph-sheet-details-permission.tsx

```diff
@@ -320,15 +335,31 @@ export function GraphSheetDetailsPermission({
                                 Weight: {s.weight}
                               </Badge>
                               <Badge variant="secondary">
-                                {calculateStreamValue(
-                                  entry.streams.reduce(
-                                    (total, s) => total + s.weight,
-                                    0,
-                                  ),
-                                  emissionsData[entry.targetAccountId],
-                                  true,
-                                  entry.targetAccountId,
-                                )}
+                                {(() => {
```

Comment:
a named function would be better in this case, to make it easier to read what is being executed

Reply:
👆
--
In general we should:

- Name intermediate values if that makes it clearer / if they are part of complicated expressions;
-  Lift all logic to the start of the component function / auxiliary functions, etc., and put as little logic as possible inside the JSX.

@Sipher